rules/group.go

```diff
@@ -511,165 +503,170 @@ func (g *Group) CopyState(from *Group) {
 // Rules can be evaluated concurrently if the `concurrent-rule-eval` feature flag is enabled.
 func (g *Group) Eval(ctx context.Context, ts time.Time) {
 	var (
-		samplesTotal atomic.Float64
-		wg           sync.WaitGroup
+		samplesTotal    atomic.Float64
+		ruleQueryOffset = g.QueryOffset()
 	)
 
-	ruleQueryOffset := g.QueryOffset()
```

Comment:
Review the `Eval` function by ignoring whitespace. I had to move the inner `eval` function